data/hai/hai missions.txt

```diff
@@ -209,6 +209,79 @@ mission "Ask the Hai about the Unfettered"
+		set "warned about hai merchant politics"
+		conversation
+			`As you sit in a spaceport bar waiting for the food that you've ordered, a Hai unceremoniously pulls up a chair opposite you and takes a seat. He looks to be a merchant, and one who has seen more than a few things besides. He squints at you before speaking, "You are Captain <last>, are you not?"`
+			choice
+				`	"That would be me."`
+					goto acknowledge
+				`	"I think you have the wrong captain."`
+			`	He slaps a device on the table, and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`
+			label acknowledge
+			`	He heaves a sigh. "You've been fighting the Unfettered, you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers, and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
+			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see, because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
+			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, and some people are less offended when you shoot down someone they've been arguing with. Everyone has friends on all sides though, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you, but the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first."`
+			`	He looks at you very seriously to make sure what he's said has sunk in.`
+			choice
+				`	"I understand that I've stepped on some toes."`
+					goto trouble
+					to display
+						"reputation: Hai" < 1
+				`	"I understand, thank you for warning me."`
+					goto end
+					to display
+						"reputation: Hai" >= 1
+				`	"How could I be in trouble for defending myself against Sympathizers?"`
+					to display
+						"reputation: Hai" < 1
+				`	"Why should I get in trouble for defending myself against Sympathizers?"`
+					to display
+						"reputation: Hai" >= 1
```

Comment:
I don't think this choice block is so complex when not using `to display` to merit its use.
```suggestion
			branch "low reputation"
				"reputation: Hai" < 1
			choice
				`	"I understand, thank you for warning me."`
					goto end
				`	"Why should I get in trouble for defending myself against Sympathizers?"`
					goto defending
			label "low reputation"
			choice
				`	"I understand that I've stepped on some toes."`
					goto trouble
				`	"How could I be in trouble for defending myself against Sympathizers?"`
			label defending
```

Reply:
Ah, good stuff. This kind of grew organically as I wrote through things, but this is much simpler yeah